run_dir/design/flowcells.html

```diff
@@ -11,7 +11,15 @@
 
 <div class="container-fluid" id="flowcell_list">
   <div id="fc_table_div">
-      <h1><span id="page_title">Flowcells</span> </h1>
+      <h1><span id="page_title">Flowcells</span>
+          {% if all %}
+            <small>Showing all flowcells</small>
+            <a class="btn btn-default" href="/flowcells">Show last 6 months only</a>
```

Comment:
```suggestion
            <a class="btn btn-primary" style="float:right;" href="/flowcells">Show last 6 months only</a>
```

Reply:
Thanks! But it doesn't look that great...
![screen shot 2019-02-12 at 17 11 58](https://user-images.githubusercontent.com/1250075/52649831-5af87200-2ee9-11e9-8b4f-fb0e5dda3227.png)

Reply:
Ah I forgot about the search bar. In fairness it was a bit of a guess on my phone whilst on a train 😉 

I thought it looked pretty good before? Maybe just stick with what you did already...